exercises/Toluwalase_Agoro/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include <iomanip>
+#include <algorithm>
+
+
+
+std::string getWord(std::ifstream& stream) {
+  /* Function accept an input stream object and return a string from
```

Comment:
Please can you move this comment above the function signature and add a couple of lines to describe the parameter and return value. This will make it follow the Doxygen convention and is described here in the Mantid coding standards:

![FunctionHeaders](https://user-images.githubusercontent.com/56295817/88275885-03f9ed80-ccd6-11ea-859a-5cd5211fd24a.PNG)